acryldata/datahub-helm
# feat(datahub-gms): Enable autoscaling via HPA

## Summary
https://github.com/datahub-project/datahub/issues/11761
The company where I work for, started crashing datahub-gms during Snowflake ingestion, and I thought it would be handy to have an autoscaler for this workload. Hence, this pull request.

## How to test it?
Edit the values.yaml of both datahub chart, and datahub-gms subchart. Make sure to enable `datahub-gms.hpa.enabled` and `global.datahub_standalone_consumers_enabled`. Then, navige to the subchart folder and:
```
cd charts/datahub/subcharts/datahub-gms
helm template test . --values values.yaml --values ../../values.yaml | yq '. | select(.kind == "HorizontalPodAutoscaler")'
```

## Checklist
- [ ] Docs related to the changes have been added/updated (if applicable)
